src/claimer.py

```diff
@@ -59,6 +59,12 @@ def claim_product(api_client, recaptcha_solution):
             'dateTo': (utc_today + dt.timedelta(days=1)).isoformat()
         }
     )
+    """Handle case when there is no Free Learning offer"""
+    offer_count = offer_response.json().get('count')
+    if offer_count == 0:
+        logger.info(" There is no Free Learning offer right now ")
+        raise Exception(" There is no Free Learning offer right now ")
```

Comment:
Can we change both messages to `"There is no Free Learning offer right now."` (no whitespaces around and ending period)?

Otherwise I agree that's better solution than current one (no handling the error at all) :+1:

Reply:
Whitespaces removed.